targets/core.minimal

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh -e
+# Copyright (c) 2014 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Internal minimal core target, just prepare the environment so that
+# other targets can run (assuming a sane chroot)
+
+REQUIRES='core.minimal'
+DESCRIPTION=''
+CHROOTBIN=''
+. "${TARGETSDIR:="$PWD"}/common"
+
+### Append to prepare.sh:
+
+if [ "${DISTROAKA:-"$DISTRO"}" = 'debian' ]; then
+    # Run dpkg in non-interactive mode
+    export DEBIAN_FRONTEND=noninteractive
```

Comment:
This should probably be in `functions`, since it just impacts the implementation of `install` and whatnot.